api/services/elastigroup/gcp/paths/group.yaml

```diff
@@ -41,14 +39,14 @@ put:
     - "Elastigroup GCP"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - $ref: "../../../../commons/parameters/groupId.yaml"
   requestBody:
     required: true
     content:
       application/json:
         schema:
           allOf:
-            - $ref: "../schemas/elastigroup.yaml"
+            - $ref: "../schemas/elastigroupUpdate.yaml"
```

Comment:
@vlad666spot those schemas looks the same to me.
please check

Reply:
fixed , elastigroupUpdate.yaml - has different required options than elastigroup.yaml